index.js

```diff
@@ -39,6 +41,17 @@ function vimeo(str) {
 }
 
 /**
+ * Get the vine id.
+ * @param {string} str - the url from which you want to extract the id
+ * @returns {string|undefined}
+ */
+function vine(str) {
+	const regex = /https:\/\/vine\.co\/v\/([a-zA-Z0-9]*)\/?/;
```

Comment:
because I am targeting all (including early) node versions, I will have to use `var` instead of `const`